statics/js/components/preference.js

```diff
@@ -73,7 +73,10 @@ var PreferenceComponent = {
         </div>\
       </div>\
       <div class="button-holder row">\
-        <button class="btn btn-lg btn-primary" type="submit">Save</button>\
+        <button class="btn btn-lg btn-primary" type="submit" title="save in local storage">Save</button>\
```

Comment:
Can this be automatic behavior of the page, so you don't need to click anything ... just add and remove filters, or change parameters and as part of the change it will be kept in 'localStorage'

Reply:
as we discussed in the call, currently added a warning for the cancel button